ParabolInc/parabol
# fix: avoid overriding custom group name with a single card

Fixes #7207

## Demo

https://user-images.githubusercontent.com/466991/196692055-d2529cbe-c823-4c7f-b651-25f820e08e21.mov

## Testing scenarios

A:

- Give a custom title to a single card group.
- Drag that card to another card (with no group title) to create a group.
- See that the custom title was transferred to a new group correctly

B:
- Smoke test other drag & drop scenarios with smart title and custom titles
